.github/workflows/manage-azure-policy.yml

```diff
@@ -42,8 +42,42 @@ jobs:
         wait: false
         scopes: |
           /subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb
-  
-  apply-azure-policy-live:
+
+  apply-azure-policy-subscription:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    strategy:
+      matrix:
+        subscriptions: [2307d175-7e49-434b-9ac2-515529b845f2]
+    runs-on: ubuntu-latest
+    env:
+      POLICY_SUBSCRIPTION: ${{ matrix.subscriptions }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Login to Azure
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZURE_CREDENTIALS}}
+        allow-no-subscriptions: false
+
+    - name: Live - Create or Update Azure Policies
+      uses: azure/manage-azure-policy@v0
+      with:
+        mode: complete
+        paths: |
+          policies/**
+          assignments/subscriptions/${{env.POLICY_SUBSCRIPTION}}/**
```

Comment:
can't you just apply to all sub folders ?

Reply:
That will work if the matrix is removed and the compliance scan step has the scopes updated instead:
`        scopes: |
          /subscriptions/2307d175-7e49-434b-9ac2-515529b845f2`

That would leave just an action job for the subscription and the management group. 

Any preference?

Reply:
do you need the compliance scan? 

the management group one already does all subscriptions under it, alternatively you can do the same just create a shell script to find all subscriptions required for the compliance scan

Reply:
Makes sense. Updated as suggested